timesketch/views/auth.py

```diff
@@ -193,8 +194,15 @@ def validate_api_token():
     if not id_token:
         return abort(HTTP_STATUS_CODE_UNAUTHORIZED, "No ID token supplied.")
 
-    client_id = current_app.config.get("GOOGLE_OIDC_API_CLIENT_ID")
-    if not client_id:
+    client_ids = []
+    client_ids.append(current_app.config.get("GOOGLE_OIDC_CLIENT_ID"))
+    additional_client_ids = current_app.config.get(
+        "WHITELISTED_GOOGLE_OIDC_API_CLIENT_IDS")
+    if additional_client_ids:
```

Comment:
you can make this a list() in the configuration instead of comma separated. The config file is just python.